pkg/domain/infra/abi/containers.go

```diff
@@ -434,6 +434,7 @@ func (ic *ContainerEngine) ContainerCheckpoint(ctx context.Context, namesOrIds [
 		TCPEstablished: options.TCPEstablished,
 		TargetFile:     options.Export,
 		IgnoreRootfs:   options.IgnoreRootFS,
+		KeepRunning:    options.LeaveRunning,
```

Comment:
My OCD is twitching that this variable name isn't "LeaveRunning", but it is what it is.